mk1282130/E-learning
# registration and list of users

## Description / Purpose (checkpoints)
This PR implements functions below.
1. register
2. login with username and password
4. register admin and user
5. display admin user list
6. display user list

Fixes issue: https://docs.google.com/spreadsheets/d/1Sm74awAFZEEztnbXPYW_foSVLmHMKcog/edit#gid=29277409
task number:1, 2, 4, 5, 6

## How To Test This PR
for user login
1. Go Login Page
2. Input User Credentials
   =>user example: [username]Tarou Yamada, [e-mail]tarou@gmail.com, [pw]testtest )
3. Click Login
4. You can see the list of users that doesn't include admins
5. Click Logout

for admin user login

6. Go Login Page
7. Input User Credentials
   =>admin user example: [username]Maki Fukudome, [e-mail]maki@gmail.com, [pw]testtest )
8. Click Login
9. You can see the list of admin users only

 **Commands**
 # php artisan migrate 

 **comment**
Design will be made later.
